DOCUMENTATION.md

```diff
@@ -1,36 +1,41 @@
 # Documentation
+
 This is the documentation of the IPCLib, the layout of this document is the following:
 - ServerSocket
 - ClientSocket
 - Setting up a connection and sending data
 
 # ServerSocket
 
-A class that represents the serverside of a socket connection. It has functions to asynchronously connect and receive data from a client, and a synchronous function to send data to a client.
+A class that represents the serverside of a socket connection. It has functions to asynchronously connect and receive data from a client and a synchronous function to send data to a client.
+
+## Initialize
+
+Initializes and starts the server.
 
 ## ConnectAsync
 
 Spawns a child thread that waits for a connection, such that the main thread is not blocked.
 
 ## AwaitClientConnection
 
-The main thread awaits untill a client has connected to the server, this makes sure that the server cannot send data to a client that does not exist. It is best practise to call this function after ConnectAsync().
+The main thread awaits untill a client has connected to the server, this makes sure that the server cannot send data to a client that is not connected. If this function is called without ConnectAsync() the main thread will connect with a client. Otherwise the main thread will wait on the sub thread until it has connected.
```

Comment:
until, not untill

Reply:
By "called without ConnectAsync()", do you mean "called before calling ConnectAsync()"? The current phrasing indicates ConnectAsync() would be passed as a parameter.